_posts/2018-04-06-all-hands.md

```diff
@@ -0,0 +1,182 @@
+- **Project templates**: developed a minimal design; **withoutboats will write an RFC**.
+- **Custom workflows**: designed workflow customization, which is useful for frameworks; **Aaron Turon will write an  RFC**.
+
+### Infrastructure
+
+- **bors queue**: hatched and resourced lots of ideas to reduce the PR queue for Rust 2018.
+- **crater**: pietroalbini is testing a bot for running crater!
+- **Travis log bot**: TimNN has written a bot to [extract errors] from Travis logs
+
+[extract errors]: https://github.com/rust-lang/rust/pull/49513#issuecomment-377538323
+
+### WG: CLI apps
+
+- [WG overview slides](https://git.io/rust-all-hands-cli).
+- **Survey and strategy**: dove into survey data and developed strategy from it; posts forthcoming.
+- **Distribution**: "distribution-friendly" badge on crates.io; [extensible Carogo install RFC](https://github.com/rust-lang/rfcs/pull/2376).
+
+### WG: network services
+
+- [WG overview slides](https://gist.github.com/withoutboats/6d4c4639b286d3da19d89d8af82d82d7).
+- **Launching the WG**: determined goals for the WG, including async/await, documentaiton, mid-level HTTP libraries, and the [Tower](https://github.com/tower-rs/tower) ecosystem.
+- **Async/await**: finalized design and stabilization approach for RFCs (&&withoutboats and Aaron Turon will write companion RFCs**).
+
+### WG: embedded devices
+
+- [WG overview slides](https://github.com/japaric/all-hands-2018-embedded)
+- **Embedded Rust on stable**: addressed all known blockers and several mid-priority issues as well.
+- **The Embedded Rust book**: defined audience and basic outline.
+
+### WG: WebAssembly
+
+- [WG overview slides](https://gist.github.com/fitzgen/700e134cffe9a8438524f0a39810a4d0).
+- **2018 edition planning**, including scoping the toolchain and book efforts for the release.
+- **JS integration**, focused on integrating JS callbacks vs Rust functions, and integrating wasm-bindgen with CommonJS.
+- **Code bloat**, reducing the footprint of panicking from 44k to 350 bytes.
+
+### Unsafe code guidelines
+
+- **Restarted the WG**: dug back into two competing approaches ("validity" and
+  "access"-based), substantially iterated on both. **Niko Matsakis and Ralf Jung
+  will be writing blog posts about these ideas**.
+- **Pressing issues**: tackled a few near-term decisions that need to be made,
+  including `union` semantics, `Pin` semantics, `thread::abort` and more.
+
+### Web site
+
+- **Domain WG sketching**: over a couple of sessions, the four domain-centered
+  working groups (listed above) developed some initial sketches of landing pages
+  for each domain.
+
+### Rust reach
+
+- **Prepared for launch**, which happened [earlier this week!](https://blog.rust-lang.org/2018/04/02/Increasing-Rusts-Reach-2018.html)
+
+### New working groups
+
+In addition to the work by existing teams, we had critical mass to form two new working groups:
```

Comment:
I'd phrase this a little less mechanical :). Critical mass reached, working group formed? :D